contracts/cw3-flex-multisig/schema/handle_msg.json

```diff
@@ -295,21 +295,21 @@
       "description": "MemberDiff shows the old and new states for a given cw4 member They cannot both be None. old = None, new = Some -> Insert old = Some, new = Some -> Update old = Some, new = None -> Delete",
       "type": "object",
       "required": [
-        "addr"
+        "key"
```

Comment:
Why the renaming? no strong feelings, but I for one like the old name better.

Reply:
I did this as I plan to make Diff generic over T so it can be used by a generic snapshot provider that will allow this pattern to be easily imported, whenever we want to keep a snapshot